kumascript/macros/LearnSidebar.ejs

```diff
@@ -168,6 +168,17 @@ var text = mdn.localStringMap({
       'Accessible_multimedia' : 'Accessible multimedia',
       'Mobile_accessibility' : 'Mobile accessibility',
       'Assessment_Accessibility_troubleshooting' : 'Assessment: Accessibility troubleshooting',
+    'Web_performance_making_websites_fast_and_responsive' : 'Web performance — Making websites fast and responsive',
+      'Web_performance_guides' : 'Web performance guides',
+        'The_why_of_web_performance' : 'The why of web performance',
+        'What_is_web_performance' : 'What is web performance',
+        'How_do_users_perceive_performance' : 'How do users perceive performance',
+        'Measuring_performance' : 'Measuring performance',
+        'Multimedia_images' : 'Multimedia images',
+        'Multimedia_video' : 'Multimedia video',
+        'JavaScript_performance_best_practices' : 'JavaScript performance best practices',
+        'HTML_performance_best_practices' : 'HTML performance best practices',
```

Comment:
It looks like we are missing [CSS performance features](https://developer.mozilla.org/en-US/docs/Learn/Performance/CSS), compared with the list under https://developer.mozilla.org/en-US/docs/Learn/Performance#guides.